src/silo/common/zstdfasta_writer.cpp

```diff
@@ -39,3 +50,14 @@ void silo::ZstdFastaWriter::writeRaw(const std::string& key, const std::string&
              << std::to_string(compressed_genome.size()) << '\n'
              << compressed_genome << '\n';
 }
+
+void silo::ZstdFastaWriter::writeDefault(const std::string& key) {
```

Comment:
What is this default sequence about? What should it be used for?

Reply:
When talking about what should happen with mutational queries on sequences, that have no aligned sequence, it was said that we should just handle them as if they were a full sequence of `N`s. Therefore I altered the ZstdSequenceWriter to be able to write a default value for a key that is provided in the constructor